Turkey/PlatformId.cs

```diff
@@ -18,7 +18,9 @@ public List<string> CurrentIds
 
         public List<string> ComputePlatformIds(string[] osReleaseLines, string lddVersionOutput)
         {
+#pragma warning disable CA1308 // Normalize strings to uppercase
```

Comment:
I've never had this warning. Do we have some specific analyzer settings?

Anyway, I'd simplify this code to (which probably won't get rid of the warning):
```cs
string arch = RuntimeInformation.OSArchitecture.ToString().ToLowerInvariant();
```
Or if you want to do something fancy:
```cs
static class EnumExtensionMethods
{
    public static string ToLowerInvariant(this Enum e)
        => e.ToString().ToLowerInvariant();
}
```